dask/bag/text.py

```diff
@@ -77,13 +79,14 @@ def read_text(path, blocksize=None, compression='infer',
 
         elif blocksize is None:
             files = open_text_files(path, encoding=encoding, errors=errors,
-                                          compression=compression, **kwargs)
+                                          compression=compression,
+                                          **storage_options)
             blocks = [delayed(list)(file) for file in files]
 
         else:
             _, blocks = read_bytes(path, delimiter=linedelimiter.encode(),
                     blocksize=blocksize, sample=False, compression=compression,
-                    **kwargs)
+                    **storage_options)
```

Comment:
This one, I think, is still OK.  These are keyword arguments that `read_bytes` genuinely expects